src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -126,6 +131,9 @@ else if (obj instanceof Block)
             return Ingredient.fromStacks(new ItemStack((Block)obj, 1, OreDictionary.WILDCARD_VALUE));
         else if (obj instanceof String)
             return new OreIngredient((String)obj);
+        else if (obj instanceof Unified)
+            //noinspection unchecked
```

Comment:
Or an annotation?

Reply:
What do you mean?

Reply:
Oh, I see I can use `@SuppressWarnings("unchecked")` if I split this up into two lines. Will do.

Reply:
I just don't know if all compilers or ides take this note.